core/src/main/java/com/xmartlabs/base/core/helper/StringUtils.java

```diff
@@ -27,17 +27,17 @@ public static boolean isNullOrEmpty(@Nullable CharSequence string) {
   }
 
   /**
-   * Changes the first letter of the given word to uppercase.
+   * Changes the first letter of the given sentence to uppercase.
    *
-   * @param word the word to capitalize
+   * @param sentence the sentence to capitalize the first letter
    * @return a new {@code string} instance with the first letter in uppercase
    */
   @NonNull
-  public static String capitalizeWord(@NonNull String word) {
-    if (word.isEmpty()) {
+  public static String capitalizeFirstLetter(@NonNull String sentence) {
```

Comment:
What about "text" instead of "sentence"? And "first char" instead of "first letter"? Because it's what it does.